<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HLK_RM04_widget</class>
 <widget class="QWidget" name="HLK_RM04_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SerialBox5_fix_baudrate" name="serial_widget" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>gate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>remote IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="IPV4_wo_port" name="gate_widget" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>encrypt type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_ssid"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>remote port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="IPV4_wo_port" name="remove_ip_widget" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>network</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>mask</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sb_remote_post"/>
     </item>
     <item row="1" column="1">
      <widget class="IPV4_wo_port" name="ip_widget" native="true"/>
     </item>
     <item row="2" column="1">
      <widget class="IPV4_wo_port" name="mask_widget" native="true"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>net_ip</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QComboBox" name="cb_encrypt_type"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="le_password"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>net_mask</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>dhcpd_ip</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>dhcpd_dns</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="IPV4_wo_port" name="net_ip_widget" native="true"/>
     </item>
     <item row="5" column="1">
      <widget class="IPV4_wo_port" name="net_mask_widget" native="true"/>
     </item>
     <item row="4" column="3">
      <widget class="IPV4_wo_port" name="dhcpd_ip_widget" native="true"/>
     </item>
     <item row="5" column="3">
      <widget class="IPV4_wo_port" name="dhcpd_dns_widget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QComboBox" name="cb_serial_to"/>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btn_serial_to">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btn_function">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="cb_function"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="LogBox" name="log_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialBox5_fix_baudrate</class>
   <extends>QWidget</extends>
   <header>serialbox5_fix_baudrate.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IPV4_wo_port</class>
   <extends>QWidget</extends>
   <header>ipv4_wo_port.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogBox</class>
   <extends>QWidget</extends>
   <header>logbox.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_ssid</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>sb_remote_post</tabstop>
  <tabstop>cb_encrypt_type</tabstop>
  <tabstop>cb_function</tabstop>
  <tabstop>btn_function</tabstop>
  <tabstop>cb_serial_to</tabstop>
  <tabstop>btn_serial_to</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
